pkg/scripts/download.go

```diff
@@ -51,6 +40,50 @@ func NewDownloader() Downloader {
 	}
 }
 
+// DownloadFile downloads the file pointed to by URL into local targetFile
+// Returns information a boolean flag informing whether any download/copy operation
+// happened and an error if there was a problem during that operation
+func (d *downloader) Download(url *url.URL, targetFile string) (bool, error) {
+	sr := d.schemeReaders[url.Scheme]
```

Comment:
use full names for variables. I hate seeing vars like 's', 'sr', 'gh' and so on. Having a proper (even long) variable name is much better than this.